src/neo/VM/Helper.cs

```diff
@@ -15,6 +15,30 @@ namespace Neo.VM
 {
     public static class Helper
     {
+        public static ScriptBuilder CreateArray<T>(this ScriptBuilder sb, IReadOnlyList<T> list = null)
+        {
+            if (list is null || list.Count == 0)
+                return sb.Emit(OpCode.NEWARRAY0);
+            for (int i = list.Count - 1; i >= 0; i--)
+                sb.EmitPush(list[i]);
+            sb.EmitPush(list.Count);
+            return sb.Emit(OpCode.PACK);
+        }
+
+        public static ScriptBuilder CreateMap<TKey, TValue>(this ScriptBuilder sb, IReadOnlyDictionary<TKey, TValue> map = null)
```

Comment:
Shouldn't we keep the naming "Emit" or "Push"?

Reply:
Because it does not directly push the map, it actually creates a new map.